### PR TITLE
Add recipes for merging files & cropping video

### PR DESCRIPTION
**Pull request by kfrn**
Two new recipes:
* _Crop video_ - in the category **Change video properties**
* _Merge audio and video tracks_, in the **Other** category, next door to _Split audio and video tracks_.

These are pretty basic commands, so let me know if you think they're in scope ... and if so, whether I've 'filed' them correctly.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -666,6 +666,36 @@ <h2 id="video-properties">Change video properties</h2>
...
+    <input type="checkbox" id="crop_video">
+    <div class="hiding">
+      <h3>Crop video</h3>
+      <p><code>ffmpeg -i <i>input_file</i> -vf "crop=<i>width</i>:<i>height</i>" <i>output_file</i></code></p>
+      <p>This command crops the input video to the dimensions defined</p>
+      <dl>
+        <dt>ffmpeg</dt><dd>starts the command</dd>
+        <dt>-i <i>input_file</i></dt><dd>path, name and extension of the input file</dd>
+        <dt>-vf "<i>width</i>:<i>height</i>"</dt><dd>Crops the video to the given width and height (in pixels).<br>
+          By default, the crop area is centred: that is, the position of the top left of the cropped area is set to x = (<i>input_width</i> - <i>output_width</i>) / 2, y = <i>input_height</i> - <i>output_height</i>) / 2.<br>
```
Comment (retokromer) [inline index.html]: The `<br>` at the end is not needed.

**Comment by retokromer** [comment]
Indeed, I think that they are in scope here!

**Review by kieranjol — COMMENTED** [review]
Definitely within scope,these are great!I always end up googling every time I have to drop usually. My only suggestion relates to the mapping in the merge command. Should "stream" be pluralised to "streams" seeing as -map 0:v maps all videos streams from first input, -map 1:a maps all audio streams from second input?

**Comment by ablwr** [comment]
Hell yeah!!!

**Comment by kfrn** [comment]
Nice catches @retokromer & @kieranjol ! I've made those changes. 🐶

**Review by retokromer — APPROVED** [review]
LGTM

**Comment by retokromer** [comment]
Thank you very much again and again and again, @kfrn!

**Comment by ablwr** [comment]
yay yaya yay

**Comment by ablwr** [comment]
I am gonna hype ffmprovisr on amia-l

**Pull request merged**